spec/css/selector.hrx

```diff
@@ -359,3 +359,119 @@ Error: unterminated attribute selector for a
 >> [a=b ï] {c: d}
 
    ---^
+
+<===>
+================================================================================
+<===> escaping/number_as_first_char_without_space/input.scss
+.\31u {clear: left;}
```

Comment:
Please [use single-letter names for irrelevant placeholders](https://github.com/sass/sass-spec/blob/master/STYLE_GUIDE.md#do-use-single-letter-names-for-irrelevant-placeholders).